exporter/containerimage/exptypes/types.go

```diff
@@ -24,3 +25,19 @@ type Platform struct {
 	ID       string
 	Platform ocispecs.Platform
 }
+
+type BuildInfo struct {
+	Type  BuildInfoType `json:"type,omitempty"`
+	Ref   string        `json:"ref,omitempty"`
+	Alias string        `json:"alias,omitempty"`
+	Pin   string        `json:"pin,omitempty"`
+}
+
+type BuildInfoType string
+
+const (
+	BuildInfoTypeImage BuildInfoType = "docker-image"
```

Comment:
Just to be aligned with source identifiers:

https://github.com/moby/buildkit/blob/f6ac37d027fc42d95bdafb66b85dcc547cd486c5/source/identifier.go#L30-L36

Reply:
Those are used as "pseudo" schemes; would it make sense to use them the same here as well?

```
git://<foo>
docker-image://<foo>
```

Or would that mean duplicate schemes, like;

```
git://http://github.com/<foobar>
```

Reply:
Ah sorry miss this comment. I think we can't because git transport can be http, git or ssh: https://github.com/moby/buildkit/pull/2311#discussion_r689467275.

Reply:
> Any reason to use `docker-image`, and not `image`

Done: https://github.com/crazy-max/buildkit/blob/6c859662340ad4e23cdb9ac6195225effac96184/exporter/containerimage/exptypes/types.go#L38-L42

Reply:
> Any reason to use docker-image, and not image

In LLB level this was left to reduce ambiguity, should more package managers be supported in the future(eg. with iso/raw images).